hackIDE/static/hackIDE/js/custom.js

```diff
@@ -411,6 +411,17 @@ $(document).ready(function(){
 							$(".error-message").html(response.run_status.stderr);
 						}
 					}
+					// To handle case when hackerearth api doesn't return a valid response.
```

Comment:
@adi8 this code block should be when ajax request fails. You have put it under `success` callback function.